### DISCUSSION TITLE
If we use cloud services, when we migrate to our own host later, do all the above data support export?

### DISCUSSION THREAD

**Discussion by wsdo** [langfuse/langfuse#973] [Support] [answered]
If we use cloud services, when we migrate to our own host later, do all the above data support export?

**Comment by marcklingen** [comment, marked as answer]
Langfuse as a project is open and does not lock you in. Without much effort you can change to a self-hosted instance at any time.

Regarding data exports, you can use the GET API to export data from a Langfuse instance (also Langfuse Cloud): https://langfuse.com/docs/api

If relevant, the Langfuse maintainers currently provide migration support for enterprise customers.
